shell/common/rasterizer.cc

```diff
@@ -77,13 +76,15 @@ void Rasterizer::Setup(std::unique_ptr<Surface> surface) {
                              user_override_resource_cache_bytes_);
   }
   compositor_context_->OnGrContextCreated();
+#if !defined(OS_FUCHSIA)
```

Comment:
Needs a flutter/flutter issue here re: getting the RasterThreadMerger working on Fuchsia

Reply:
Done.